examples/demo.js

```diff
@@ -213,10 +183,12 @@ window.onload = function() {
     onLoad: function onFileReaderLoad(progress) {
       var reader = progress.target;
       var step = sequencer.images.image1.steps[0];
+      var util=IntermediateHtmlStepUi(sequencer);
       step.output.src = reader.result;
       sequencer.run({ index: 0 });
       step.options.step.imgElement.src = reader.result;
-      updatePreviews(reader.result);
+      updatePreviews(reader.result,'addStep');
```

Comment:
Would this only run if there are some shown? And just checking, does this code all run on the final output or the original input? Maybe it ought to run on the final output; the last step before the add interface... what do you think?